microsoft/PowerToys
# [localization] move file to its correct location

## Summary of the Pull Request

**What is this about:**
This commit https://github.com/microsoft/PowerToys/commit/99397799323e55409b268a4fdd8c55eb5d9a34f4 misplaced the resource file for Settings.UI for the Japanese translation.

**What is include in the PR:** 
moved the misplaced file to its correct destination:
from `src/core/Microsoft.PowerToys.Settings.UI/loc/ja/src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us`
to `src/settings-ui/Microsoft.PowerToys.Settings.UI/loc/ja/src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us`

**How does someone test / validate:** 
Run a farm build and verify that the Settings app is correctly translated to Japanese.

## Quality Checklist

- [x] **Linked issue:** https://github.com/microsoft/PowerToys/issues/8905

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
